bundles/org.openhab.binding.tibber/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -47,7 +47,7 @@
 		</channels>
 		<properties>
 			<property name="vendor">Tibber</property>
-			<property name="thingTypeVersion">2</property>
+			<property name="thingTypeVersion">3</property>
```

Comment:
This is not needed, since there are no update instructions provided. Update instructions are only needed when modifying channels.